scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1351,11 +1504,8 @@ public void removeThirdPartyCredential(String key) throws NotConnectedException,
                                        params.isPending(),
                                        params.isRunning(),
                                        params.isFinished());
-        List<TaskId> lTaskId = new ArrayList<TaskId>(pTaskInfo.getList().size());
-        for (TaskInfo taskInfo : pTaskInfo.getList()) {
-            lTaskId.add(taskInfo.getTaskId());
-        }
-        return new Page<TaskId>(lTaskId, pTaskInfo.getSize());
+        return new Page<TaskId>(pTaskInfo.getList().stream().map(TaskInfo::getTaskId).collect(Collectors.toList()),
+                                pTaskInfo.getSize());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)